### DISCUSSION TITLE
Line plot tidak come out the way it supposed to. Capstone DV

### DISCUSSION THREAD

**Discussion by mynugraErlangga** [teamalgoritma/community#713] [Q&A] [answered]
Saya sudah mencontoh code chunk untuk membuat a lineplot dari  inclass.
Plot yang keluar sbb:
![Screenshot 2023-07-03 235551](https://github.com/teamalgoritma/community/assets/125937571/7115bd9c-37e3-4d78-9b1a-63c6cc121ede)
![Screenshot 2023-07-03 235606](https://github.com/teamalgoritma/community/assets/125937571/463bdf3e-8876-42fb-90cd-c6c4c2f3d009)

Attached is my .Rmd and data .csv used
[GithubQ_ErlanggaN_CapstoneDV.zip](https://github.com/teamalgoritma/community/files/11949624/GithubQ_ErlanggaN_CapstoneDV.zip)

Desired output:
A normal looking lineplot with lines connecting the dots, not like the picture previously shown.

**Comment by ranydwi** [comment, marked as answer]
Dear @mynugraErlangga 

Terima kasih sudah bersedia menunggu. Untuk plot yang bapak tampilkan di atas, sepertinya untuk codenya ada yang kurang Pak. Silahkan tambahkan code berikut untuk memasukan `TYPE` sebagai pemisah antara garis-garisnya.
```
geom_line(aes(group = TYPE)) +
```
atau 

```
geom_line(aes(color = TYPE)) +
geom_point(aes(color = TYPE))
```

Semoga membantu.
Regards.
